### DISCUSSION TITLE
Steve Krouse - val.town

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#153] [Announcements]
## Background

- https://twitter.com/stevekrouse
- https://stevekrouse.com/
- https://val.town
- https://futureofcoding.org/episodes/

### Background

- [x] You built "the coding school" and after school program for kids. Why did you found it and was it successful?
- [x] What was Zaplib and why didn't it work? Do you think that's changed?
- [x] You had a lunch with Alan Kay. How dod it change you perspective on how you approach your work?

### Val.town

- [x] Why did you start val.town?
- [x] Where did the idea come from?
- [x] Why do you think end user programming is important?
- [x] What do you think sets val apart from services like repl.it, observable, etc?
- [x] Are there any limitations built into the platform?
- [x] What are some unique features to val town that others don't have? (API access, cron)
- [x] What are some cool things your community has built?
- [x] Can you give us a glimpse at how you plan to monetize?
- [x] What's the long term vision? What are you building towards?
- [x] Will Val Town ever support other languages?

### Future

- [x] What does the future of programming look like?

**Comment by just-be-dev** [comment]
Tooltips:
- https://www.nushell.sh/
- https://typst.app/

**Comment by hipstersmoothie** [comment]
- [Ninja Creami](https://www.amazon.com/Ninja-NC501-Milkshakes-Programs-Containers/dp/B0B9CZ6XBQ/ref=sr_1_5?hvadid=541713554845&hvdev=c&hvlocphy=9032176&hvnetw=g&hvqmt=e&hvrand=4140134316454350828&hvtargid=kwd-1603941546952&hydadcr=7466_9611873&keywords=ninja+cream+ice+cream+maker&qid=1680116027&sr=8-5)
- https://github.com/ktock/container2wasm

**Comment by stevekrouse** [comment]
Tooltips:

* https://tweethunter.io/twemex
